plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/CodeAssist.java

```diff
@@ -271,11 +280,10 @@ public void documentChanged(DocumentEvent event) {
                     p.apply(document);
                 }
 
-                ProposalApplyResult result = dtoFactory.createDto(ProposalApplyResult.class);
                 result.setChanges(changes);
                 Point selection = p.getSelection(document);
                 if (selection != null) {
-                    Region region = dtoFactory.createDto(Region.class);
+                    Region region = DtoFactory.newDto(Region.class);
```

Comment:
What do you think about using with method? It will be looks like
```
                     result.setSelection(DtoFactory.newDto(Region.class).withOffset(selection.x)
                                                                                                           withLength(selection.y));
```